module/hooks/chat.mjs

```diff
@@ -117,10 +117,21 @@ function triggerRollStep(argString) {
 
 function addUserPortrait( msg, jquery ) {
 
-  const avatar = msg.user.avatar;
+  const chatAvatarSetting = game.settings.get( "ed4e", "chatAvatar" );
+  let avatar;
+  if ( chatAvatarSetting === "configuration" && game.user.isGM ) {
```

Comment:
Is the `isGM` necessary? If setting's scope is `client` every player can decide for themselves?